<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="Text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>50</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancel">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>217</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ok">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>217</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="Tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>322</width>
      <height>168</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ABC">
     <attribute name="title">
      <string>ABC</string>
     </attribute>
     <widget class="QPushButton" name="Language">
      <property name="geometry">
       <rect>
        <x>203</x>
        <y>99</y>
        <width>114</width>
        <height>45</height>
       </rect>
      </property>
      <property name="text">
       <string>English</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterI">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>24</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterT">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>24</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>t</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CapsLock">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>99</y>
        <width>62</width>
        <height>45</height>
       </rect>
      </property>
      <property name="text">
       <string>Caps Lock</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Shift">
      <property name="geometry">
       <rect>
        <x>59</x>
        <y>99</y>
        <width>58</width>
        <height>45</height>
       </rect>
      </property>
      <property name="text">
       <string>â†‘</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterY">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>24</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>y</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterQ">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>24</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>q</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterE">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>24</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>e</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterR">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>24</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>r</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterU">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>24</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>u</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Space">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>99</y>
        <width>90</width>
        <height>45</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterW">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>24</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>w</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterO">
      <property name="geometry">
       <rect>
        <x>199</x>
        <y>24</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>o</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterP">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>24</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>p</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSTilde">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>~</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterA">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>49</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>a</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterS">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>49</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterD">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>49</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>d</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterF">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>49</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>f</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Number2">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Number3">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Number4">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Number5">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Number6">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Number7">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Number8">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Number9">
      <property name="geometry">
       <rect>
        <x>199</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Number0">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterG">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>49</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>g</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterH">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>49</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>h</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterJ">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>49</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>j</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterK">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>49</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>k</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterL">
      <property name="geometry">
       <rect>
        <x>199</x>
        <y>49</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>l</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterZ">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>74</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>z</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterX">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>74</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Delete">
      <property name="geometry">
       <rect>
        <x>274</x>
        <y>-1</y>
        <width>43</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NewLine">
      <property name="geometry">
       <rect>
        <x>249</x>
        <y>24</y>
        <width>68</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>â¤¶</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterC">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>74</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>c</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterV">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>74</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>v</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterB">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>74</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>b</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterN">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>74</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>n</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterM">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>74</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>m</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSap">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>49</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>'</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSequ">
      <property name="geometry">
       <rect>
        <x>249</x>
        <y>49</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSDash">
      <property name="geometry">
       <rect>
        <x>249</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSvb">
      <property name="geometry">
       <rect>
        <x>274</x>
        <y>49</y>
        <width>43</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSComma">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>74</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>,</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSPoint">
      <property name="geometry">
       <rect>
        <x>199</x>
        <y>74</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSQuestionMark">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>74</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSExclamationMark">
      <property name="geometry">
       <rect>
        <x>249</x>
        <y>74</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSat">
      <property name="geometry">
       <rect>
        <x>274</x>
        <y>74</y>
        <width>43</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>@</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Number1">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSPoundSign">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Â£</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetterSEuro">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>-1</y>
        <width>27</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>â‚¬</string>
      </property>
     </widget>
     <zorder>LetterSEuro</zorder>
     <zorder>LetterSPoundSign</zorder>
     <zorder>Language</zorder>
     <zorder>LetterI</zorder>
     <zorder>LetterT</zorder>
     <zorder>CapsLock</zorder>
     <zorder>Shift</zorder>
     <zorder>LetterY</zorder>
     <zorder>LetterQ</zorder>
     <zorder>LetterE</zorder>
     <zorder>LetterR</zorder>
     <zorder>LetterU</zorder>
     <zorder>Space</zorder>
     <zorder>LetterW</zorder>
     <zorder>LetterO</zorder>
     <zorder>LetterP</zorder>
     <zorder>LetterSTilde</zorder>
     <zorder>LetterA</zorder>
     <zorder>LetterS</zorder>
     <zorder>LetterD</zorder>
     <zorder>LetterF</zorder>
     <zorder>Number2</zorder>
     <zorder>Number3</zorder>
     <zorder>Number4</zorder>
     <zorder>Number5</zorder>
     <zorder>Number6</zorder>
     <zorder>Number7</zorder>
     <zorder>Number8</zorder>
     <zorder>Number9</zorder>
     <zorder>Number0</zorder>
     <zorder>LetterG</zorder>
     <zorder>LetterH</zorder>
     <zorder>LetterJ</zorder>
     <zorder>LetterK</zorder>
     <zorder>LetterL</zorder>
     <zorder>LetterZ</zorder>
     <zorder>LetterX</zorder>
     <zorder>Delete</zorder>
     <zorder>NewLine</zorder>
     <zorder>LetterC</zorder>
     <zorder>LetterV</zorder>
     <zorder>LetterB</zorder>
     <zorder>LetterN</zorder>
     <zorder>LetterM</zorder>
     <zorder>LetterSap</zorder>
     <zorder>LetterSequ</zorder>
     <zorder>LetterSDash</zorder>
     <zorder>LetterSvb</zorder>
     <zorder>LetterSComma</zorder>
     <zorder>LetterSPoint</zorder>
     <zorder>LetterSQuestionMark</zorder>
     <zorder>LetterSExclamationMark</zorder>
     <zorder>LetterSat</zorder>
     <zorder>Number1</zorder>
    </widget>
    <widget class="QWidget" name="More">
     <attribute name="title">
      <string>More</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Symbol">
     <attribute name="title">
      <string>Symbol</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Mobile">
     <attribute name="title">
      <string>Mobile</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="I_Forgot">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>217</y>
      <width>124</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>I Forgot</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
